docs/source/performance/cache-backends.mdx

```diff
@@ -201,6 +201,24 @@ const server = new ApolloServer({
 });
 ```
 
+## Handling cache fetching errors
+When cache backend is unavailable, all requests depend on it will fail. It is crucial to still return successful response in this kind of scenario.
+
+To provide error tolerance for cache backends that connect via a client (e.g., Redis), we recommend using the `ErrorsAreMissesCache` wrapper from [@apollo/utils.keyvaluecache](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#errorsaremissescache). If the cache is unavailable and your request throws an error, `ErrorsAreMissesCache` treats that error as a cache miss. At the same time, your cache client can keep trying to reconnect to your cache backend until things are working again, like so:
+
+An example usage looks like this:
```

Comment:
I'd suggest removing this sentence and the extra line above it (so the previous sentence ending in "like so:" is directly above the example code block): 
```suggestion
```